models/user.js

```diff
@@ -0,0 +1,27 @@
+const mongoose = require('mongoose');
+const Song = require('./song');
```

Comment:
Is this used?

Reply:
No, it's actually not, I implemented it as part of trying to route the ObjectID from the songs but it doesn't seem to be functional at this time.